src/rgw/rgw_sal_motr.cc

```diff
@@ -5365,7 +5393,7 @@ int MotrStore::next_query_by_name(string idx_name,
       next_key = dir + "\xff"; // skip all dir content in 1 step
     else
       next_key = key_out[i + k - 1] + " ";
-    ldout(cctx, 0) <<__func__ << ": do_idx_next_op(): next_key=" << next_key << dendl;
+    ldout(cctx, 20) <<__func__ << ": do_idx_next_op(): next_key=" << next_key << dendl;
```

Comment:
Please update this to `log_debug` as well.

Reply:
there are 116 places where this should updated. We can have separate to make that change.

Reply:
I am okay with both, changing as we go along or changing once for all with a separate ticket.